### ISSUE TITLE
Show shorthand hex

### ISSUE THREAD

**Issue by lol768** [bionicrm/Color#5]
You might already ~~does~~ do this, but "#aabbcc" should become "#abc"

**Comment by tmazeika** [comment]
Yep, hex editing is messed up in a few ways at the moment; see #6 as well.
